mods/multitest/depends.txt

```diff
@@ -1,3 +0,0 @@
-default

```

Comment:
Why remove depends ?

Reply:
Didn't see a mod.conf

Reply:
`mod.conf` wasn't deleted, it's still there; check it out at the `update_3d_armor` branch.